evalml/automl/automl_algorithm/default_algorithm.py

```diff
@@ -90,6 +90,7 @@ def __init__(
         n_jobs=-1,
         text_in_ensembling=False,
         top_n=3,
+        ensembling=True,
```

Comment:
Should we set this false to match the behavior in AutoMLSearch? 

What I don't like about this is that we change the user parameter value silently for time series if it's not `False`.  Perhaps it would be better to raise an exception if ensembling is set to true for time series?

Reply:
will do!